README.md

```diff
@@ -59,23 +59,58 @@ The library comprises two packages:
 1. Create a custom layout template for your pages using `hmrcLayout` as per the section
       [HMRC layout](#hmrc-layout)
 
-1.  Optionally, add `TwirlKeys.templateImports` in `build.sbt`:
-    ```sbt
-        TwirlKeys.templateImports ++= Seq(
-          "uk.gov.hmrc.govukfrontend.views.html.components._",
-          "uk.gov.hmrc.govukfrontend.views.html.helpers._",
-          "uk.gov.hmrc.hmrcfrontend.views.html.components._",
-          "uk.gov.hmrc.hmrcfrontend.views.html.helpers._"
-        )
-    ```
-
-    Adding this removes the need for `@import` statements in your Twirl templates.
-    If you prefer not to use this mechanism, import the components in any template that uses them as
-    follows:
-
-    ```scala
-    @import uk.gov.hmrc.hmrcfrontend.views.html.components._
-    ```
+1. Then to use our components and helpers in your templates, you will need to import the component or helper from its
+   package `uk.gov.hmrc.(govuk|hmrc)frontend.views.html.(components|helpers)`.
+
+   For most components the parameters that can be provided are encapsulated in a viewmodel, comprised of case classes
+   that live within a subpackage of `uk.gov.hmrc(govuk|hmrc)frontend.views.viewmodels` and are aliased for use
+   under `uk.gov.hmrc(govuk|hmrc)frontend.views.html.components`.
+
+   ```scala
+   /*
+   Import all components, viewmodels, and their implicits. Most succinct import method, but may require additional 
```

Comment:
I'm not sure this is correct. The components._ would not bring in any implicit classes as far as I can see.

Reply:
```suggestion
   Import all components and viewmodels. Most succinct import method, but may require additional 
```

Reply:
refactored! I have included helpers as well even though they're not used to the first example so it's clearer what's needed to be able to pull in everything.